app/views/admin/attachments/index.html.erb

```diff
@@ -1,46 +1,47 @@
+  } %>
+<% end %>
 
-<div class="row">
-  <section class="col-md-8">
-    <h1>Attachments for <%= attachment.attachable_model_name %></h1>
-
-    <%= attachable_editing_tabs(attachable) do %>
-      <p class="qa-helper-copy">
-        <strong>Note:</strong>
-        <%= attachment_note(attachment.attachable_model_name) %>
-      </p>
-      <ul class="actions list-unstyled">
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <ul class="govuk-list">
+      <li>
+        <%= link_to 'Upload new file attachment', new_polymorphic_path([:admin, typecast_for_attachable_routing(attachable), Attachment]), class: "govuk-link govuk-link--no-visited-state" %>
+      </li>
+      <% if attachable.is_a?(Edition) %>
+        <li>
+          <%= link_to 'Bulk upload from Zip file'.html_safe, new_admin_edition_bulk_upload_path(attachable), class: "govuk-link govuk-link--no-visited-state" %>
+        </li>
+      <% end %>
+      <% if attachable.allows_html_attachments? %>
+        <li>
+          <%= link_to 'Add new HTML attachment', new_polymorphic_path([:admin, typecast_for_attachable_routing(attachable), Attachment], type: "html"), class: "govuk-link govuk-link--no-visited-state" %>
+        </li>
+      <% end %>
+      <% if attachable.allows_external_attachments? %>
         <li>
-          <%= link_to 'Upload new file attachment', new_polymorphic_path([:admin, typecast_for_attachable_routing(attachable), Attachment]) %>
+          <%= link_to 'Add new external attachment', new_polymorphic_path([:admin, typecast_for_attachable_routing(attachable), Attachment], type: "external"), class: "govuk-link govuk-link--no-visited-state" %>
         </li>
-        <% if attachable.is_a?(Edition) %>
-          <li>
-            <%= link_to 'Bulk upload from Zip file'.html_safe, new_admin_edition_bulk_upload_path(attachable) %>
-          </li>
-        <% end %>
-        <% if attachable.allows_html_attachments? %>
-          <li>
-            <%= link_to 'Add new HTML attachment', new_polymorphic_path([:admin, typecast_for_attachable_routing(attachable), Attachment], type: "html") %>
-          </li>
-        <% end %>
-        <% if attachable.allows_external_attachments? %>
-          <li>
-            <%= link_to 'Add new external attachment', new_polymorphic_path([:admin, typecast_for_attachable_routing(attachable), Attachment], type: "external") %>
-          </li>
-        <% end %>
-      </ul>
+      <% end %>
+    </ul>
 
-      <div class="alert alert-warning">
-        <p>
-          You must upload attachments in an <a href="https://www.gov.uk/guidance/content-design/planning-content#open-formats">open standards format</a>.<br/>
-        </p>
-        <p>
-          For example, if an attachment is text-based and designed to be edited it should be uploaded to GOV.UK as an .odt (OpenDocument text) file instead of a closed format like .docx.
-        </p>
+    <details class="govuk-details" data-module="govuk-details">
```

Comment:
I think we can use the details component here so any changes to the design system will be applied when the publishing components gem is updated

https://components.publishing.service.gov.uk/component-guide/details


```
  <%= render "govuk_publishing_components/components/details", {
      title: " An important note regarding uploading attachments",
    } do %>
      <p>You must upload attachments in an open standards format. </p>
      <p>For example, if an attachment is text-based and designed to be edited it should be uploaded to GOV.UK as an .odt (OpenDocument text) file instead of a closed format like .docx.</p>
      <p>Read more about <%= link_to "open standards formats", "https://www.gov.uk/guidance/content-design/planning-content#open-formats", class: "govuk-link govuk-link--no-visited-state" %>.</p>
    <% end %>
```